Source/WebCore/html/parser/ParsingUtilities.h

```diff
@@ -225,7 +225,7 @@ template<typename CharacterType> bool skipExactlyIgnoringASCIICase(StringParsing
 
     if (buffer.lengthRemaining() < literalLength)
         return false;
-    if (!equalLettersIgnoringASCIICase(buffer.position(), literal))
+    if (!equalLettersIgnoringASCIICaseWithLength(buffer.span(), literal.span8(), literalLength))
```

Comment:
Can this use `buffer.span().first(lliteralLength)`?

Reply:
Are you arguing for calling equalLettersIgnoringASCIICase() instead of equalLettersIgnoringASCIICaseWithLength() by taking sure the first 2 parameters have the same length?

The issue is that this wouldn't be as efficient:
- There would be the extra cost of calling `first(literalLength)` on the first parameter
- equalLettersIgnoringASCIICase() would start by making sure that the 2 input spans have the same length (which would be redundant).

FYI, the pattern I used in this PR is that same one I had used in my earlier PR for equalIgnoringASCIICase() + equalIgnoringASCIICaseWithLength().

Reply:
OK